packages/fxa-settings/src/components/GetDataTrio/en.ftl

```diff
@@ -3,6 +3,7 @@
 get-data-trio-title-firefox = { -brand-firefox }
 get-data-trio-title-firefox-recovery-key = { -brand-firefox } account recovery key
 get-data-trio-title-firefox-backup-verification-codes = { -brand-firefox } backup authentication codes
+get-data-trio-title-backup-verification-codes = Backup authentication codes
```

Comment:
In our emails, we refer to these codes as "backup authentication codes" without a brand reference. I confirmed via Slack we want to be consistent and not refer to a brand here. I also saw `get-data-trio-title-firefox` was used as a "fallback" but shouldn't ever be used, so I believe we can safely remove that fallback logic and don't need the string updated. I can add `get-data-trio-title-mozilla = { -brand-mozilla }` though if we we want to cover it anyway.

I didn't change the recovery key references in this file because they appear to be old strings that are not used with the new recovery key flow.

Reply:
Should we delete them if they are no longer used?

Reply:
@dschom Missed your comment. It's referenced in-code as a fallback for a switch statement, so I think we'd want to remove that and then that fallback switch string could be removed. I can do this or file a 1-pointer for it.